.ci_support/linux_mpimpichpython3.6.yaml

```diff
@@ -0,0 +1,34 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '7'
+channel_sources:
+- conda-forge,defaults
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- gxx
```

Comment:
Shouldn't these compiler names correspond to the MPI wrappers ?

Reply:
These are automatically generated and outside of our control.  I guess that's what I would have expected but I don't have much expertise.

Reply:
Note: This is also the case for `hdf5`: https://github.com/conda-forge/hdf5-feedstock/blob/master/.ci_support/linux_mpimpich.yaml#L10